test/scripts/e2e_go_tests.sh

```diff
@@ -81,42 +82,20 @@ cd ${SRCROOT}/test/e2e-go
 
 # ARM64 has some memory related issues with fork. Since we don't really care
 # about testing the forking capabilities, we're just run the tests one at a time.
-EXECUTE_TESTS_INDIVIDUALLY="false"
+PARALLEL_FLAG=""
 ARCHTYPE=$("${SRCROOT}/scripts/archtype.sh")
 echo "ARCHTYPE:    ${ARCHTYPE}"
 if [[ "${ARCHTYPE}" = arm* ]]; then
-    EXECUTE_TESTS_INDIVIDUALLY="true"
+    PARALLEL_FLAG="-p 1"
 fi
 
-echo "EXECUTE_TEST_INDIVIDUALLY = ${EXECUTE_TESTS_INDIVIDUALLY}"
+echo "PARALLEL_FLAG = ${PARALLEL_FLAG}"
 
 if [ "${#TESTPATTERNS[@]}" -eq 0 ]; then
-    if [ "${EXECUTE_TESTS_INDIVIDUALLY}" = "true" ]; then
-        TESTS_DIRECTORIES=$(GO111MODULE=off go list ./...)
-        for TEST_DIR in ${TESTS_DIRECTORIES[@]}; do
-            TESTS=$(go test -list ".*" ${TEST_DIR} -vet=off | grep -v "github.com" || true)
-            for TEST_NAME in ${TESTS[@]}; do
-                go test ${RACE_OPTION} -timeout 1h -vet=off -v ${SHORTTEST} -run ${TEST_NAME} ${TEST_DIR} | logfilter
-                KMD_INSTANCES_COUNT=$(set +o pipefail; ps -Af | grep kmd | grep -v "grep" | wc -l | tr -d ' ')
-                if [ "${KMD_INSTANCES_COUNT}" != "0" ]; then
-                    echo "One or more than one KMD instances remains running:"
-                    ps -Af | grep kmd | grep -v "grep"
-                    exit 1
-                fi
-                ALGOD_INSTANCES_COUNT=$(set +o pipefail; ps -Af | grep algod | grep -v "grep" | wc -l | tr -d ' ')
-                if [ "${ALGOD_INSTANCES_COUNT}" != "0" ]; then
-                    echo "One or more than one algod instances remains running:"
-                    ps -Af | grep algod | grep -v "grep"
-                    exit 1
-                fi
-            done
-        done
-    else
-        go test ${RACE_OPTION} -timeout 1h -v ${SHORTTEST} ./... | logfilter
-    fi
+    gotestsum --format pkgname --jsonfile integrationtestresults.json -- ${RACE_OPTION} ${PARALLEL_FLAG} -timeout 1h -v ${SHORTTEST} ./...
```

Comment:
running it on ARM64 used to consistently fail many of the e2e tests due to a bug in the fork implementation. I don't know how running it using gotestsum resolves the issue - but I'm ok with that.
In this case, use the `GOTESTCOMMAND` variable you've defined, and set the `EXECUTE_TESTS_INDIVIDUALLY` to false before going into the branching if `GOTESTCOMMAND` is using the gotestsum